HIPs/hlf-operator.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: Hyperledger Fabric Operator Proposal
+parent: Hyperledger Improvement Proposals
+---
+
+# HIP Identifier
+[HIPs/hlf-operator.md](./hlf-operator.md)
+
+# Sponsor(s)
+
+- David Viejo [dviejo@kfs.es](mailto:dviejo@kfs.es)
+- Baohua Yang [baohua.yang@oracle.com](mailto:baohua.yang@oracle.com)
+- Aditya Joshi [adityaprakashjoshi1@gmail.com](mailto:adityaprakashjoshi1@gmail.com)
+
+# Abstract
+
+HLF Operator is a Kubernetes operator that helps simplify the deployments of Hyperledger Fabric in Kubernetes. It is based on the Hyperledger Fabric Helm charts and provides a Kubernetes native way to deploy and manage Hyperledger Fabric networks.
+
+# Context
+
+There are many ways to deploy Hyperledger Fabric, but most of the alternatives focus on Docker and manually performed deployments, not to mention there's no support to perform operations like:
+
+- Renewal of the certificates
+- Upgrades
+- Channel management
+
+HLF-operator solves this by introducing the concept of the Kubernetes operator, which is a program that runs within Kubernetes and extends it with functionality specific to HLF that makes managing networks way easier than other alternatives such as Docker-based.
```

Comment:
Have you considered talking to [Hyperledger Cello](https://github.com/hyperledger/cello) maintainers and see how does the project differ from them?

Reply:
The main difference is:

- HLF-operator doesn't depend on a database, only on the Kubernetes cluster.
- Their operator that runs on Kubernetes is outdated (2020); maybe they could use it too.
- Cello focuses on docker + Kubernetes, while hlf-operator only on Kubernetes
- Cello offers a user interface to manage the nodes, while hlf-operator doesn't

Reply:
Thanks for the differentiation. It will be nice to capture these in the proposal. We could also have one common meeting with the maintainers of both projects to understand if Cello can start using this operator.